matlab/nfct/FFTW_ESTIMATE.m

```diff
@@ -0,0 +1,23 @@
+% FFTW_ESTIMATE FFT flag
+%   Valid for FFTW3
+%
+%   Copyright (c) 2002, 2016 Jens Keiner, Stefan Kunis, Daniel Potts
```

Comment:
Ooops, need to update the year in the copyright notice to 2017. But I will check that anyway when I create the next release, so no need to change here.